### ISSUE TITLE
Sudoku Solver Using CNN,RNN and GAN model

### ISSUE THREAD

**Issue by ashis2004** [Niketkumardheeryan/ML-CaPsule#971]
## Description:
Sudoku is a popular logic-based number puzzle. This project demonstrates how to use Convolutional Neural Networks (CNN) with TensorFlow to solve Sudoku puzzles by recognizing and filling in the missing numbers.

## The primary objective of this project is to:
Develop a CNN, RNN and GAN model and find accuracy
@invigorzz313 pls assign it to me I'm GSSOC'24 contributor

**Comment by Vishisht16** [comment]
Hi, I can create a model for this. Please assign it to me.
